src/components/Step.vue

```diff
@@ -88,9 +99,18 @@ export default class Step extends Vue {
     };
   }
 
+  deleteThisStep() {
```

Comment:
We can call it simply `delete` as it's a method of a Step component

Reply:
I chose this naming as delete() is reserved and deleteStep() is already taken by the mutation